dict/portuguese.yml

```diff
@@ -0,0 +1,291 @@
+    'KR': [Coreia do Sul, Coréia do Sul, República da Coreia, Coreia, Sul-Corean*, Seul]
+    'MN': [Mongólia, Mongol*, Mongólic*, Ulã Bator, Ulan Bator]
+    'MO': [Macau, Macauense*, Macaense*, Macaíst*]
+    'TW': [Taiwan, Taiwanês*, Formosan*, Formosin*, Taiuanês*, Taipé, Cidade de Taipé]
+
+  SOUTH:
+    'AF': [Afeganistão, Afegã*, Afegan*, Afegânic*, Cabul, Cábul]
+    'BD': [Bangladesh, Bangladexe, Bangladeche, Bangladês*, Banglades*, Bangladense*, Bengal*, Bangladexense*, Bangladexian*, Bangladechian*, Bangladeshian*, Bengalês, Bengalense*, Bengala*, Bengali*, Daca, Dhaka]
+    'BT': [Butão, Butanês*, Butanense*, Butani*, Butâni*, Timbu, Thimphu, Timpu, Thimbu]
+    'IN': [Índia, Indian*, Bombaim, Mumbai, Nova Deli, Nova Delhi, Nova Déli]
+    'IR': [Irã, Irão, Iranian*, Teerã, Teerão]
+    'LK': [Sri Lanka, Sri Lanca, Seri Lanca, Ceilão, Cingalês*, Srilankês*, Srilanquês*, Colombo, Columbo]
+    'MV': [Maldivas, Maldiv*, Maldívi*]
+    'NP': [Nepal, Nepalês, Nepales*, Catmandu]
+    'PK': [Paquistão, Paquistanês*, Islamabade, Islamabad]
+
+  SOUTH-EAST:
+    'BN': [Brunei, Brunean*, Bruneín*]
+    'ID': [Indonésia, Indonési*, Jacarta]
+    'KH': [Camboja, Cambodja, Cambojan*, Phnom Penh, Pnom Pen, Pnom Pene, Penome Pene]
+    'LA': [Laos, Laosian*, Laocian*, Vientiane, Vienciana]
+    'MM': [Myanmar, Birmânia, Mianmarense*, Birmanês*, Birmã*, Birman*, Birmanian*, Birmanense*, Rangum, Yangon, Nepiedó, Naypyidaw, Nay Pyi Taw]
+    'MY': [Malásia, Malaio*, Malaia*, Malási*, Kuala Lumpur, Cuala Lumpur, Quala Lumpur, Cualalampur, Putrajaia, Putrajaya]
+    'PH': [Filipinas, Filipin*,  Manila]
+    'SG': [Singapura, Cingapura, Singapurense*, Singapuran*]
+    'TH': [Tailândia, Tailandês*, Banguecoque, Bangkok, Bancoque]
+    'TL': [Timor-Leste, Timorense*, Maubere*, Díli]
+    'VN': [Vietnã, Vietname, Vietnamit*, Vietnamês*, Vietnamense*, Vietnamian*, Hanói, Ho Chi Minh, Saigon, Saigão]
+
+  WEST:
+    'AE': [Emirados Árabes Unidos, EAU, Emiradense*, Emirátic*, Dubai, Abu Dhabi, Abu Dabi]
+    'AM': [Arménia, Armênia, Arméni*, Armêni*, Erevã, Erevan, Ierevan, Ierevã]
+    'AZ': [Azerbaijão, Azeri*, Bacu, Baku]
+    'BH': [Bahrein, Barém, Barein, Bareine, Bareinit*, Baremit*, Baremense*, Baremês*, Manama]
+    'CY': [Chipre, Cipriot*, Cípri*, Ciprense*, Nicósia]
+    'GE': [Geórgia, Georgian*, Geórgic*, Tiblíssi, Tbilisi]
+    'IL': [Israel, Israelit*, Israelense*, Jerusalém]
+    'IQ': [Iraque, Iraquian*, Bagdá, Bagdade]
+    'JO': [Jordânia, Jordanian*, Jordan*, Amã]
+    'KW': [Kuwait, Kuwaitian*, Koweitian*, Cuwaitian*, Coveitian*, Kowaitian*, Cuvait*, Kuwait*, Cuait*, Koweit, Coveite, Couaite, Quaite]
+    'LB': [Líbano,  Libanês*, Beirute]
+    'OM': [Omã, Omão, Omanense*, Omanian*, Omani, Mascate]
+    'PS': [Palestina, Palestin*, Gaza, Cisjordânia]
+    'QA': [Catar, Qatar, Catarense*, Catari*, Catarian*, Doha, Doa]
+    'SA': [Arábia Saudita, Saudit*, Riade]
+    'SY': [Síria, Siríac*, Sírio*, Síria*, Damasco]
```

Comment:
Síria* should match Síria

What about Síri* to have less items in the dictionary, would this create too many false positives?

Reply:
I did some search and does not seem like it would be a problem. I've changed.